osgar/drivers/system_monitor.py

```diff
@@ -36,9 +64,19 @@ def run_input(self):
 
             if self.platform == "linux":
                 temp_raw = psutil.sensors_temperatures()
-                assert "acpitz" in temp_raw, temp_raw
-                temp = temp_raw["acpitz"][0].current  # Expects only one CPU.
-                self.bus.publish('temp', temp)
+                if "acpitz" in temp_raw:
+                    temp = temp_raw["acpitz"][0].current  # Expects only one CPU.
+                elif "k10temp" in temp_raw:
+                    temp = temp_raw["k10temp"][0].current  # Expects only one CPU.
+                else:
+                    temp = None  # Unsupported case.
+                if temp is not None:
+                    self.bus.publish('temp', temp)
+                proc_dmesg = subprocess.Popen(["dmesg"], stdout=subprocess.PIPE)
```

Comment:
so the process is opened always even if `self.log_dmesg` is false?

Reply:
what about 
```
 --since time
           Display record since the specified time. The time is possible
           to specify in absolute way as well as by relative notation
           (e.g. '1 hour ago'). Be aware that the timestamp could be
           inaccurate and see --ctime for more details.
```
https://man7.org/linux/man-pages/man1/dmesg.1.html